includes/class-sensei-course.php

```diff
@@ -1155,50 +1155,13 @@ public function course_lesson_preview_count( $course_id = 0 ) {
 	 */
 	public function get_product_courses( $product_id = 0 ) {
```

Comment:
This should have a `@deprecated` tag.

Reply:
https://github.com/Automattic/sensei/pull/2495